WS2012R2/lisa/setupscripts/RemoveIsoFromDvd.ps1

```diff
@@ -87,13 +88,30 @@ $error.Clear()
 #
 # Remove the .iso file from the VMs DVD drive
 #
+
+#
+#Get the version of VM
+#
+$vmGeneration = Get-VM $vmName | select -ExpandProperty Generation
```

Comment:
Please use the -ComputerName param for Get-VM.
This is required if the VM is on a remote server.